### ISSUE TITLE
register and login on the front-page

### ISSUE THREAD

**Issue by szabgab** [kantord/LibreLingo#1790]
IMHO it would be a good idea to have a link to register and to log in on the front-page as well and also indicated if someone is already logged in on the front-page as well.

**Comment by kantord** [comment]
Good idea. Though I think it's also nice that you can try it out immediately and decide to register later

Also, when someone is already logged in, I imagine they should actually not even see the home page. Rather they should be immediately redirected to the course they are learning, no?

**Comment by szabgab** [comment]
I agree on both. What if someone logs out or visits with a different browser possible from another device. That person will see the same page as the totally new person but will probably want to go and log in right away.

**Comment by kantord** [comment]
>  That person will see the same page as the totally new person but will probably want to go and log in right away.

This is a very good point. Also, there will always be users who just want to sign up even before starting a course, so this needs to be included on the some page

**Comment by frostedkitsune** [comment]
to implement these in order, we need a sitemap first do we have that?
